torchci/pages/metrics.tsx

```diff
@@ -560,6 +562,14 @@ export default function Page() {
               queryParams={[]}
               badThreshold={(value) => value > 60 * 60 * 6} // 6 hours
             />
+            <ScalarPanel
+              title={"# flaky tests"}
```

Comment:
i feel like a better name overall would be '# disabled tests' because tests can be disabled for reasons other than flakiness.  If we only want flaky tests, we should probably search for the label "module: flaky-tests"

Reply:
Calling this `# disabled tests` is indeed much clearer, and this is what we want to track